src/web/routes/application/common/populate-session.test.js

```diff
@@ -0,0 +1,30 @@
+const test = require('tape')
+const sinon = require('sinon')
+const { populateSession } = require('./populate-session')
+
+test('populateSession() should copy fields from the request body by default', (t) => {
```

Comment:
Should this be testing `defaultTransform()` directly, and the `populateSession()` test just assert that the default function is called?